src/Views/Component.elm

```diff
@@ -59,18 +60,43 @@ addButton { addLabel, db, items, openSelectModal, scope } =
         , class "gap-1 w-100"
         , id "add-new-element"
         , disabled <| List.isEmpty availableComponents
-        , onClick <| openSelectModal autocompleteState
+        , onClick <| openSelectComponentModal autocompleteState
         ]
         [ i [ class "icon icon-plus" ] []
         , text addLabel
         ]
 
 
-componentView :
-    Config db msg
-    -> ( Component.Quantity, Component, List ExpandedElement )
-    -> Component.Results
-    -> List (Html msg)
+addElementTransformButton : Config db msg -> Component -> Int -> Html msg
+addElementTransformButton { db, openSelectTransformModal } component index =
+    let
+        availableTransformProcesses =
+            db.processes
+                |> Process.listByCategory Category.Transform
+                |> Process.available
+                    (component.elements
+                        |> LE.getAt index
+                        |> Maybe.map .transforms
+                        |> Maybe.withDefault []
```

Comment:
What about sorting by `displayName` then?

Reply:
Addressed in https://github.com/MTES-MCT/ecobalyse/pull/967/commits/70c97030d692a1f5b6d0349e98d9a89a5273b7bb